### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by mravinale**
* removed unused tasks
* removed tools
* added BE reload support
* added BE watch reload

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,19 +2,12 @@
   "name": "react-es6-redux-boilerplate",
   "version": "1.0.0",
   "scripts": {
-    "prestart": "babel-node tools/startMessage.js",
     "start": "npm-run-all --parallel open:src lint:watch test:watch",
-    "open:src": "babel-node tools/srcServer.js",
+    "open:src": "babel-watch api/srcServer.js",
```
Comment (juampick) [inline package.json:6]: ;)

File: api/srcServer.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+app.use(require('webpack-dev-middleware')(compiler, {
+  noInfo: true,
+  publicPath: config.output.publicPath
+}));
+
+app.use(require('webpack-hot-middleware')(compiler));
+
+// get all posts
+app.get('/api/myPosts', function (req, res) {
```
Comment (juampick) [inline api/srcServer.js:26]: Great

File: src/actions/postActions.js [hunk 3]
```diff
@@ -8,11 +7,13 @@ export function loadPostSuccess(posts){
 
 export function loadPosts(){
     return dispatch => {
-        return blogAPI.getAllPosts().then(posts => {
-            dispatch(loadPostSuccess(posts));
-        }).catch(error => {
-            throw(error);
-        })
+
+        return fetch('/api/myPosts')
```
Comment (juampick) [inline src/actions/postActions.js:11]: Awesome, is working as planned 👍

File: src/api/mockBlogApi.js [hunk 4]
```diff
@@ -1,91 +0,0 @@
-import delay from './delay';
```
Comment (juampick) [inline src/api/mockBlogApi.js:1]: Not sure about removing this file until we have all methods on the api. What do you think @mravinale ?

Comment (juampick) [reply src/api/mockBlogApi.js:1]: If not, when this merged, I can rebase my save/creation branch, and add the method there...

File: src/routes.js [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
+import {Route, IndexRoute,Redirect } from 'react-router';
 import App from './components/App';
 import HomePage from './components/home/HomePage';
 import AboutPage from './components/about/AboutPage';
 import BlogPage from './components/blog/BlogPage';
 import NotFound from './components/not_found/NotFound';
 
 export default (
-    <Route path="/" component={App}>
+    <Route path="/app" component={App}>
```
Comment (juampick) [inline src/routes.js:10]: Liked this redirection. So now we have the `/api` and `/app` really neat. You are the man!!! 💯

File: src/actions/postActions.js (hunk 3 above)
Comment (mravinale) [reply src/actions/postActions.js:11]: :)

**Comment by mravinale** [comment]
thanks for the corrections 👍

**Pull request merged**